### DISCUSSION TITLE
torch_geometric OSError

### DISCUSSION THREAD

**Discussion by fffahim** [pyg-team/pytorch_geometric#2771] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45016119/123286702-f5dcce80-d52f-11eb-92f3-78f8a4037728.png)
How to solve this issue?

**Comment by wsad1** [comment]
Check if deleting you pip cache and reinstalling helps. As described in point 2 [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html#id1).

**Comment by rusty1s** [comment, marked as answer]
Colab comes with PyTorch 1.9.0 by default, so you need to update as well, see [here](https://colab.research.google.com/drive/1h3-vJGRVloF5zStxL5I0rSy4ZUPNsjy8?usp=sharing):
```
!pip install -q torch-scatter -f https://pytorch-geometric.com/whl/torch-1.9.0+cu102.html
!pip install -q torch-sparse -f https://pytorch-geometric.com/whl/torch-1.9.0+cu102.html
!pip install -q torch-geometric
```
